src/curated-corpus/pages/CustomSectionsPage/CustomSectionsPage.tsx

```diff
@@ -0,0 +1,488 @@
+        const end = section.endDate ? new Date(section.endDate) : null;
+        const hasStarted = !!start && start <= now;
+        const hasEnded = !!end && end < now;
+
+        if (section.disabled || hasEnded) {
+          grouped.expired.push(section);
+          return;
+        }
+
+        // If start date is in the future, it's scheduled regardless of active status
+        if (start && start > now) {
+          grouped.scheduled.push(section);
+          return;
+        }
+
+        // Treat active=true as Live (unless disabled/expired/future above)
+        if (section.active) {
+          grouped.live.push(section);
+          return;
+        }
+
+        if (hasStarted) {
+          grouped.live.push(section);
+          return;
+        }
+
+        grouped.scheduled.push(section);
+      });
+
+      // Sort each group by start date (most recent first)
+      Object.keys(grouped).forEach((key) => {
+        grouped[key as keyof GroupedSections].sort((a, b) => {
+          const dateA = a.startDate ? new Date(a.startDate).getTime() : 0;
+          const dateB = b.startDate ? new Date(b.startDate).getTime() : 0;
+          return dateB - dateA;
+        });
+      });
+
+      setGroupedSections(grouped);
+    }
+  }, [data]);
+
+  const getStatusChip = (section: Section) => {
+    const status = section.status as SectionStatus | null;
+
+    if (status === SectionStatus.Disabled) {
+      return (
+        <Chip
+          label="Disabled"
+          size="small"
+          sx={{ backgroundColor: '#F3F3F5', color: '#6B6B7B', fontWeight: 500 }}
+        />
+      );
+    }
+
+    if (status === SectionStatus.Expired) {
+      return (
+        <Chip
+          label="Expired"
+          size="small"
+          sx={{ backgroundColor: '#FFF0F0', color: '#D32F2F', fontWeight: 500 }}
+        />
+      );
+    }
+
+    if (status === SectionStatus.Scheduled) {
+      return (
+        <Chip
+          label="Scheduled"
+          size="small"
+          sx={{ backgroundColor: '#F3F3F5', color: '#6B6B7B', fontWeight: 500 }}
+        />
+      );
+    }
+
+    if (status === SectionStatus.Live) {
+      return (
+        <Chip
+          label="Live"
+          size="small"
+          sx={{ backgroundColor: '#F0F7FF', color: '#1976D2', fontWeight: 500 }}
+        />
+      );
+    }
+
+    // Fallback
+    return (
+      <Chip
+        label="Unknown"
+        size="small"
+        sx={{ backgroundColor: '#F3F3F5', color: '#6B6B7B', fontWeight: 500 }}
+      />
+    );
+  };
+
+  const renderSectionTable = (title: string, sections: Section[]) => {
+    if (sections.length === 0) return null;
+
+    return (
+      <Box mb={4} sx={{ width: '100%', maxWidth: '100%' }}>
+        <Typography
+          variant="h5"
+          sx={{
+            color: '#222222',
+            fontFamily:
+              'Graphik Web, proxima-nova, Helvetica Neue, Helvetica, helvetica, arial, sans-serif',
```

Comment:
similar to the comment above, I don't think we need to set these directly here, the style/fonts are defined in `theme.ts` which get applied globally